H_ecto_models.md

```diff
@@ -214,24 +214,18 @@ defmodule HelloPhoenix.User do
     field :bio, :string
     field :number_of_pets, :integer
 
-    timestamps
+    timestamps()
   end
 
-  @required_fields ~w(name email bio number_of_pets)
-  @optional_fields ~w()
-
   @doc """
-  Creates a changeset based on the `model` and `params`.
-
-  If no params are provided, an invalid changeset is returned
-  with no validation performed.
+  Builds a changeset based on the `struct` and `params`.
   """
-  def changeset(model, params \\ :empty) do
-    model
-    |> cast(params, @required_fields, @optional_fields)
+  def changeset(struct, params \\ %{}) do
+    struct
+    |> cast(params, [:name, :email, :bio, :number_of_pets])
```

Comment:
@ybur-yug We should perhaps note (below when we are describing `cast/3`) that only fields that exist in the schema can be in the list of fields in the `cast/3` function.